Makefile

```diff
@@ -29,3 +32,6 @@ test-cover:
 	go test -v ./... --coverprofile=cover.out
 
 release: compile_frontend static_assets compile_backend
+
+deploy:
```

Comment:
Please rename this one to `deploy-kube` or something as just `deploy` is too ambiguous and could mean anything.

Reply:
Also, might be prudent to add a dry-run option.